src/routes/organizations.js

```diff
@@ -8,4 +9,9 @@ router.get('/search', async (req, res) => {
   res.json(resp);
 });
 
+router.get('/read', async (req, res) => {
```

Comment:
Since we read only one record and it will be based on its unique ID, so this will be `router.get('/read/:id', async (req, res) => {`